README.md

```diff
@@ -154,13 +155,25 @@ qrencode -t ANSI256 'otpauth://totp/VIP%20Access:VSSTXXXX?secret=YYYY&issuer=Sym
 
 Scan the code into your TOTP generating app, like Authy.
 
+
+### Using HOTP
+
+Use VSMB as token type `vipaccess provision -p -t VSMB`
+Known Limitations:
+* No way to re-synchronize if token ever gets out of sync with VIP Server.
```

Comment:
Does the official Symantec app have a way to do this?

Reply:
URL mentioned [here](https://support.symantec.com/en_US/article.TECH29053.html ) no longer works, as most of the VIP tokes are now time based. Out of sync tokens are simply corrected by having the right time. I am not sure about the official app. Since its time based having the right time would be enough to sync it with server.

Reply:
I'm a bit confused by this. What is the use of the HOTP tokens?

If the official Symantec app doesn't support them… how do any sites actually support them?

Reply:
Sorry for the confusion. I meant there is no way to re-sync the tokens. I have the token with Norton account and it works perfectly. I have checked it with Google Authenticator as well as programming one of the slots of the Yubikey and the otp code works fine with all apps/services which support VIP.